src/views/RequestRepository/index.tsx

```diff
@@ -134,11 +227,166 @@ const RequestRepository = () => {
     return 'none';
   };
 
+  const csvHeaders = [
```

Comment:
Can we refactor this into a separate file so it doesn't muddy this one? Something we can do after merge...